client/src/pages/CheckinPage.tsx

```diff
@@ -40,9 +40,9 @@ const CheckinPage: FC = () => {
   };
 
   return (
-    <div className="CheckinPage">
+    <div>
       <Stack spacing={2}>
-        <Typography variant={"h5"}>{deskId}</Typography>
+        <Typography className="headerText">{deskId}</Typography>
```

Comment:
Please add this example also to the task about styling in MUI, because I have remarks on this